.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - 3.6
+before_install:
+  - python --version
+  - pip install -U pip
+  - pip install -U pytest
+  - pip install codecov
+  - pip install flake8
+install:
+  - pip install -r requirements.txt
+  - pip install -r dev-requirements.txt
+script: 
+    # stop the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude=harvester_ng/csw,tests/csw,harvester_ng/datajson/tests_with_ckan
```

Comment:
why are we excluding harvester_ng/csw?

Reply:
Because we didn't apply last refactor to CSW (just applied to DataJSON) and it include referentes to import that do not exist anymore.

Reply:
Please turn off lint on a file by file level (in the file itself, e.g. using the `#noqa` comment) instead.

Reply:
Changed [here](https://github.com/datopian/ckan-ng-harvest/commit/07e4c42c695de562a88f8bfc2957a123d16963b3)